crawlerr/models/__init__.py

```
@@ -0,0 +1,43 @@
+    # RAG models
+    "RagQuery",
+    "RagResult",
+    "SearchMatch",
+    "DocumentChunk",
+    "EmbeddingResult",
+    # Source models
+    "SourceInfo",
+    "SourceType",
+    "SourceFilter",
+    "SourceMetadata",
+]
```

Ruff (0.12.2)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)